README.md

```diff
@@ -82,6 +82,7 @@ DEFENDER_RELAY_API_URL=
 DEFENDER_RELAY_POOL_ID=
 DEFENDER_RELAY_POOL_CLIENT_ID=
 # admin client
+DEFENDER_API_V2_URL=
```

Comment:
Should we maybe leave a comment that the URL is same as ADMIN? Or is it in docs somewhere?

Reply:
Is not in docs, I will add a comment for that

Reply:
Just curious but why have two different ENV variables for the same thing? Is this for backwards compatibility?